File: php-slim/lib/Model/ResponseFaresBreakdownItem.php
<?php
/**
 * ResponseFaresBreakdownItem
 */
namespace OpenAPIServer\Model;

/**
 * ResponseFaresBreakdownItem
 */
class ResponseFaresBreakdownItem
{
    
    /** @var \OpenAPIServer\Model\ResponseTransportationMode[] $modes */
    private $modes;
    
    /** @var int[] $routePartIds */
    private $routePartIds;
    
    /** @var \OpenAPIServer\Model\ResponseFareTicket[] $tickets */
    private $tickets;
}

File: scala-akka-http-server/src/main/scala/org/openapitools/server/model/ResponseFaresBreakdownItem.scala
package org.openapitools.server.model


/**
 * @param modes  for example: ''null''
 * @param routePartIds  for example: ''null''
 * @param tickets  for example: ''null''
*/
final case class ResponseFaresBreakdownItem (
  modes: Seq[ResponseTransportationMode],
  routePartIds: Seq[Int],
  tickets: Seq[ResponseFareTicket]
)

